austation/code/modules/admin/verbs/kill_rads.dm

```diff
@@ -0,0 +1,22 @@
+var/static/list/notradkillable = typesof(/obj/item/twohanded/required/fuel_rod) + typesof(/obj/machinery/atmospherics/components/trinary/nuclear_reactor)
```

Comment:
please for god sake don't define this as a global without even using the global defines LMAO
maybe put the static inside the proc itself so it's shared across all instances

Reply:
whoops